protocol/src/main/java/io/zeebe/protocol/record/intent/MessageStartEventSubscriptionIntent.java

```diff
@@ -16,13 +16,9 @@
 package io.zeebe.protocol.record.intent;
 
 public enum MessageStartEventSubscriptionIntent implements Intent {
-  OPEN((short) 0),
-  OPENED((short) 1),
-
-  CORRELATED((short) 4),
-
-  CLOSE((short) 2),
-  CLOSED((short) 3);
+  CREATED((short) 0),
```

Comment:
One thing we might want to consider here is whether we want to reuse the old ordinals. E.g. if you set Correlated to 4 and deleted to 3, we could read more of the old records. Not sure there is a point though, as there are ordinals of the old records which we cannot read anymore anyway.

Another option is to keep all the old ordinals, reassign the reused ones, deprecate the ones we won't use anymore. Then we could write out exceptions when we encounter an unsupported ordinal.

Again, not sure it is worth it.

Reply:
Usually, I would agree. But since the version will not be compatible anyway, we don't need to care.